apps/web/public/index.html

```diff
@@ -26,6 +26,20 @@
     -->
     <title>Novu Manage Platform</title>
     <script src="%PUBLIC_URL%/env-config.js"></script>
+    <!-- Google Tag Manager -->
+    <script>
+      (function (w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : '';
+        j.async = true;
+        j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, 'script', 'dataLayer', 'GTM-KXMC4XP2');
```

Comment:
@LetItRock 

As per [Justin's message in slack](https://novu.slack.com/archives/C06L46Z0NUA/p1713561664279229?thread_ts=1713553101.438379&cid=C06L46Z0NUA), it will be the same for all environments.

+ Added conditions and support of environment variables. I can remove these conditions and variables if you think it is not needed anymore.